src/testdir/test_spellrare.vim

```diff
@@ -0,0 +1,71 @@
+" Test spell checking
+" Note: this file uses latin1 encoding, but is used with utf-8 encoding.
```

Comment:
This file is ASCII-only, and is recognized as utf-8 by Vim, so this comment doesn't really make sense.

Reply:
That'll teach me to copy existing test files :-/